src/backend/distributed/commands/distribute_object_ops.c

```diff
@@ -837,6 +865,16 @@ GetDistributeObjectOps(Node *node)
 			return &Any_CreateFunction;
 		}
 
+		case T_CreateFdwStmt:
+		{
+			return &Any_CreateFdw;
```

Comment:
Since foreign data wrappers are almost always created by an extension and we explicitly do not want to propagate objects created by an extension, we could skip CREATE FOREIGN DATA WRAPPER.

Reply:
What if it's not created by an extension? Like the one we have in the regression tests:

```sql
CREATE FUNCTION fake_fdw_handler()
RETURNS fdw_handler
AS 'citus'
LANGUAGE C STRICT;

CREATE FOREIGN DATA WRAPPER fake_fdw HANDLER fake_fdw_handler;
```

I know that it doesn't seem like something a user would do :) 
So you're suggesting to skip those CREATE FDW statements because (almost) no one will try to create fdw's manually, right?

Reply:
The utility seems low and there is some cost (e.g. making sure we don't accidentally break anything, making sure deparsing logic does not have SQL injection bugs). I'm ok supporting it, but I'm also ok not supporting it.